tests/foreman/ui/test_errata.py

```diff
@@ -163,37 +169,88 @@ def vm(module_repos_collection_with_setup, rhel7_contenthost, target_sat):
     return rhel7_contenthost
 
 
+@pytest.fixture
+def registered_contenthost(
+    rhel_contenthost,
+    module_org,
+    module_lce,
+    module_target_sat,
+    repos=[CUSTOM_REPO_URL],
+):
+    """RHEL ContentHost registered in satellite,
+    Using SCA and global registration.
+
+    :param repos: list of upstream URLs for custom repositories,
+        default to CUSTOM_REPO_URL
+    """
+    content_view = module_target_sat.api.ContentView(
+        organization=module_org,
+        environment=[module_lce],
+    ).create()
+
+    activation_key = module_target_sat.api.ActivationKey(
+        organization=module_org,
+        environment=module_lce,
+    ).create()
+
+    custom_repos = []
+    for repo_url in repos:
+        rhel_contenthost.create_custom_repos(custom_repo=repo_url)
```

Comment:
I don't think we should need to create custom repos at the host side as we should consume the repos through the Satellite.

Reply:
I had issues during registration (error for no enabled repos) before using this method, I'll try again without. Could have been related to some other part of setup.